components/welcome_post_renderer/index.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import {useDispatch, useSelector} from 'react-redux';
+
+import {Post} from '@mattermost/types/posts';
+import {isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import PostMarkdown from 'components/post_markdown';
+import {submitCommand} from 'actions/views/create_comment';
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/common';
+import {PostDraft} from 'types/store/draft';
+import {localizeMessage} from 'utils/utils';
+
+import './index.scss';
+
+export default function WelcomePostRenderer(props: {post: Post}) {
+    const isAdmin = useSelector(isCurrentUserSystemAdmin);
+    const currentChannelId = useSelector(getCurrentChannelId);
+    const {formatMessage} = useIntl();
+
+    const dispatch = useDispatch();
+
+    let message = '';
+    const actions: React.ReactNode[] = [];
+
+    const makeOnClickForCommand = (command: string) => {
+        return () => {
+            const draft: PostDraft = {
+                message: command,
+                uploadsInProgress: [],
+                fileInfos: [],
+            };
+            dispatch(submitCommand(currentChannelId, '', draft));
+        };
+    };
+
+    const makeButton = (text: React.ReactNode, command: string) => {
+        return (
+            <button
+                onClick={makeOnClickForCommand(command)}
+            >
+                {text}
+            </button>
+        );
+    };
+
+    const helpButton = makeButton(formatMessage({
+        id: 'welcome_post_renderer.button_label.slash_help',
+        defaultMessage: '/help',
+    }), '/help');
+
+    if (isAdmin) {
+        message = [
+            '### ' + localizeMessage('welcome_post_renderer.admin_message.title', 'Welcome to Mattermost! 🚀'),
```

Comment:
Not sure if we add the unicode emojis as part of the translated messages.
Maybe better to add it as the md emoji markup `:rocket:` > :rocket: